charts/capsule/Chart.yaml

```diff
@@ -21,7 +21,7 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.9
+version: 0.1.10
```

Comment:
Please, don't bump the chart, we're going to build a new version in a foreseeable short time.